src/frontend/src/env/networks.btc.env.ts

```diff
@@ -14,3 +14,9 @@ export const BITCOIN_CANISTER_IDS: Record<MinterCanisterIdText, CanisterIdText>
 		[IC_CKBTC_MINTER_CANISTER_ID]: 'ghsi2-tqaaa-aaaan-aaaca-cai'
 	})
 };
+
+export const BTC_MAINNET_ENABLED =
+	JSON.parse(import.meta.env.VITE_BITCOIN_MAINNET ?? false) === true;
+
+export const BTC_UNIVERSE_ENABLED =
```

Comment:
I don't use - on purpose - the term "universe" in Oisy. I guess you can use `NETWORK`?

Reply:
changing it